### DISCUSSION TITLE
Unable to get past "Your application cannot be reviewed until you fix the following:"

### DISCUSSION THREAD

**Discussion by kden** [community/community#52087] [GitHub Education]
I haven't been able to get past the following on the page https://education.github.com/discount_requests/10082066/additional_information

> Your application cannot be reviewed until you fix the following:
> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
> You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

- I've just updated my regular GitHub bio with my full name and a short bio.  I hope that's what they mean.
- I'm in Wisconsin but the school is in Michigan.  I have a student ID card, but any other documentation I have is electronic.  I only have electronic documentation for this.  Why can't I email or upload documents?  Why can you only take pictures?  What kind of information do you need to recognize that I'm a student, and how do I send it?

**Comment by samus-aran** [comment]
Hi @kden 👋 

How did you get on? We also ask that when you update your profile to log out and log back in again before re-applying.
It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it.

**Comment by kden** [comment]
I haven't been able to use it with IntelliJ.  I haven't had a lot of time
to try it out, but when I did try it, it says I've successfully logged into
GitHub and then kicks me out a few seconds later over and over again.  Does
it have to match the license I'm using for IntelliJ?  I have an Ultimate
license of my own, but initially I tried it on a different installation
which was not mine.

Also, are there limits to the languages it covers?  Can it help with Kotlin?

**Comment by kden** [comment]
I think I was not sure what you were asking.  I was able to submit my application, but I had to use the mobile app.  There seem  to be issues with the desktop app.  More info here: https://github.com/orgs/community/discussions/52088#discussioncomment-5556401
